Assets/MRTK/SDK/Features/UX/Scripts/Cursors/FingerCursor.cs

```diff
@@ -63,65 +63,61 @@ protected override void UpdateCursorTransform()
                     indexFingerRotation = transform.rotation;
                 }
 
+                //Debug.Log(indexFingerPosition);
+                Debug.Log(indexFingerRotation);
+
                 Vector3 indexKnucklePosition;
                 if (!TryGetJoint(TrackedHandJoint.IndexKnuckle, out indexKnucklePosition, out _)) // knuckle rotation not used
                 {
                     indexKnucklePosition = transform.position;
                 }
 
-                if (!nearPointer.IsInteractionEnabled)
-                {
-                    // If the pointer is disabled, make sure to turn the ring cursor off
-                    // but still want show the proximity effect on bounding content
-                    if (indexFingerRingRenderer != null)
-                    {
-                        UpdateVisuals(indexFingerRingRenderer, 1, false);
-                    }
-                }
-                else if (nearPointer.IsNearObject)
+                float distance = float.MaxValue;
+                Vector3 surfaceNormal = Vector3.zero;
+
+                bool surfaceNormalFound = false;
+                bool showVisual = true;
+
+                //Show the cursor if we are deemed to be near an object or if it is near a grabbable object
+                if (nearPointer.IsNearObject)
```

Comment:
Shouldn't IsNearObject return true if a grab pointer is near a grabbable? It's a little odd that if the object is not near a grabbable, we check if the pointer is near a grabbable.

Reply:
It's the isNearObject of the pokepointer, which is different from the grab pointer. This bit of unintuitiveness is why we need to separate out cursors and pointers 😅😅

Reply:
Can you clarify that in the comment? I didn't realize it was the poke pointer.